### DISCUSSION TITLE
Newest compute shaders API

### DISCUSSION THREAD

**Discussion by e2002e** [Kode/Kore#886] [Q&A]
Hello here, I have been working with compute shaders for armory3d, but it will soon use the newest API.
I need a doc for this if possible.
Thanks in advance.

**Comment by RobDangerous** [comment]
Oh hey, it's you. After you told me that I can die in the hospital I decided not to help you anymore. Let's see if I can find a block button...

**Reply by luboslenco** [reply]
Duh, sorry to hear that, hope all is well. Long live Robert!

**Reply by RobDangerous** [reply]
Thanks! It's all right. Broke my foot but it looks like I'll finally be allowed to walk again next week.
